File: src/main/java/cub/hackerrank/tutorials/a30DaysOfCode/day09/Solution.java
package cub.hackerrank.tutorials.a30DaysOfCode.day09;

import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

class Solution {
    public static void main(String[] argh) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        Map<String, Integer> pb = new HashMap<>();
        for (int i = 0; i < n; i++) {
            String name = in.next();
            Integer phone = in.nextInt();
            pb.put(name, phone);
        }
        while (in.hasNext()) {
            String s = in.next();
            System.out.println(pb.containsKey(s)
                ? s + "=" + pb.get(s).toString()
                : "Not found"
            );
        }
        in.close();
    }
}

File: src/main/cub/hackerrank/tutorials/a30DaysOfCode/day09/Solution.kt
package cub.hackerrank.tutorials.a30DaysOfCode.day09

import java.util.*

fun main(args: Array<String>) {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT. */
    val scanner = Scanner(System.`in`)
    val n = scanner.nextInt()
    val pb: MutableMap<String, Int> = HashMap()
    for (i in 0 until n) {
        val name = scanner.next()
        val phone = scanner.nextInt()
        pb[name] = phone
    }
    while (scanner.hasNext()) {
        val s = scanner.next()
        println(
            if (pb.containsKey(s)) s + "=" + pb[s].toString() else "Not found"
        )
    }
    scanner.close()
}
